htdocs/societe/contact.php

```diff
@@ -133,6 +133,27 @@
 	include DOL_DOCUMENT_ROOT.'/core/actions_changeselectedfields.inc.php';
 }
 
+if ($action == 'confirm_delete') {
```

Comment:
If we have a test on action, we must also have a test on a permission here
`if ($action = '...' && ...permission...)`